h/session.py

```diff
@@ -89,6 +89,10 @@ def _group_model(route_url, group):
         model['url'] = route_url('group_read',
                                  pubid=group.pubid,
                                  slug=group.slug)
+
+    if user is not None and user.id == group.creator_id:
+        model['is_moderator'] = True
```

Comment:
I'd also like to hear some input from @robertknight, @sean-roberts, or @sheetaluk on omitting this field if the user isn't a moderator of the group. My inclination would be to return `"moderator": false` in those cases, as it simplifies the work of any API client.